### PR TITLE
[#705] Update documentation to indicate no IAM user role needed to run the app

### LINKED ISSUE #705: Update documentation to indicate no IAM user role needed to run the app
**Context:**
As per a discussion group question, we noticed it was unclear that iAm permissions were not required to start up the app. 
[Discussion group thread](https://groups.google.com/g/cloud-carbon-footprint/c/Ft-VvHDhIMI/m/xKTf1iiPAgAJ)
"So actually to run the app, you do not need to create a new IAM User so we should actually update that documentation."

**A/C's:**
- [ ] Update wording that may be misleading around when it is necessary to create a new IAM User  [here](https://www.cloudcarbonfootprint.org/docs/aws))
"You will need an [IAM](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/) user that can create access-keys and modify your billing settings." should not be there.

### PR DESCRIPTION
**Pull request by ashutoshkrris**
## Description of Change
This PR fixes #705 and updates the documentation to indicate no IAM user role is needed to run the app

Write a brief but concise description of the changes you have made while also making sure to **link the relevant issue number to this PR** _( e.g. fixes #ISSUE-NUMBER - optional short description)_

## Checklist

- [x] PR description included and stakeholders cc'd
- [ ] tests are changed or added
- [ ] yarn test passes
- [ ] yarn lint has been run
- [ ] git pre-commit hook is successfully executed.
      (**Please refrain from using `--no-verify`**)
- [ ] yarn changeset was run and relevant packages have been updated as a major/minor/patch bump with descriptions
      (**Determine version update using [Semantic Versioning](https://semver.org/)**)
- [ ] relevant documentation is changed or added

## Notes

Additional information relevant to the changes

© 2021 Thoughtworks, Inc.

### CODE REVIEW

File: microsite/docs/ConnectingData/AWS.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Your AWS account needs to be configured to generate Cost and Usage reports and s
 
 1.  Ensure your aws account has the correct permissions
 
-    - You will need an [IAM](https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/) user that can create access-keys and modify your billing settings.
+    - IAM permissions are not required to run the app.
```
Comment (camcash17) [inline microsite/docs/ConnectingData/AWS.md]: What we want to convey here is that you do not need to create a new IAM user or access keys, you just need an IAM role that will give you access to to Billing data via the CUR and Athena.

I think it is still fine to remove this link, but we may want to change the documentation to reflect the key point here.

**Review by camcash17 — CHANGES_REQUESTED** [review]
Hi @ashutoshkrris, thanks for the PR! Please take a look at my comment.

File: microsite/docs/ConnectingData/AWS.md (hunk 1 above)
Comment (ashutoshkrris) [reply microsite/docs/ConnectingData/AWS.md]: Hi @camcash17 
Please check now

**Review by 4upz — APPROVED** [review]
The new message LGTM! Thanks for contributing @ashutoshkrris

**Pull request merged**